index.html

```diff
@@ -27,17 +27,119 @@ <h1>I'm Saningo Glad to see you</h1>
         <section class="social-details">
           <h5>LET'S CONNECT</h5>
           <div class="social-wrapper">
-            <span class="social-icon"><img src="/images/Vector.svg" alt="social-icon"/></span>
-            <span class="social-icon"><img src="./images/Vector.png" alt="social-icon" /><img alt="social-icon" src="/images/Vect-1.png"/></span>
-            <span class="social-icon"><img src="./images/Vect-1.svg" alt="social-icon"/></span>
-            <span class="social-icon"><img src="./images/Vect-2.svg" alt="social-icon" /></span>
-            <span class="social-icon"><img src="./images/Vect-3.svg" alt="social-icon" /></span>
+            <ul>
+              <li class="social-icon"><img src="/images/Vector.svg" alt="social-icon"/></li>
+              <li class="social-icon"><img src="./images/Vector.png" alt="social-icon" /><img alt="social-icon" src="/images/Vect-1.png"/></li>
+              <li class="social-icon"><img src="./images/Vect-1.svg" alt="social-icon"/></li>
+              <li class="social-icon"><img src="./images/Vect-2.svg" alt="social-icon" /></li>
+              <li class="social-icon"><img src="./images/Vect-3.svg" alt="social-icon" /></li>
+            </ul>
           </div>
         </section>
+        <!-- Projects Section -->
+        <section class="work">
+          <div class="card">
+            <div class="card-container">
+              <div class="card-1"></div>
+              <div class="card-desc">
+                <h2>Tonic</h2>
+                <ul>
+                  <li>CANOPY</li>
+                  <li>Back End Dev</li>
```

Comment:
- [ ] You did an amazing job implementing cards but as previous reviewer said to fix the issue with cards kindly do it following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Here you have screenshots of how I am seeing it:

![image](https://user-images.githubusercontent.com/74506933/153671604-557dc9e9-ade7-4165-a7db-b7bb39529ed5.png)

![image](https://user-images.githubusercontent.com/74506933/153671630-165cea91-6d61-4c40-81f7-d7de488cf9af.png)

Reply:
- [ ] [REQUIRED] Kindly note that you need to stick to the requirement as much as possible. Please refer to my comment above regarding the cards section.